pages/common/jar.md

```diff
@@ -0,0 +1,7 @@
+# jar
+
+> Tool empowering work with *.jar files, used to distribute application software or libraries on the Java platform.
```

Comment:
Maybe leave the asterisk out? Github seems to format the rest of the line as italics.

Also "Tool empowering work with ..." sounds a bit grandiose. Maybe change to something simpler/more direct?